.orchestra/support/fetch.sh

```diff
@@ -19,10 +22,21 @@ while [[ $# -gt 0 ]]; do
         shift # past argument
         shift # past value
         ;;
+        --hash)
+        # Format is <hash type>-<hash>, for example sha512-0123456789abcdef...
+        HASH_CMD=$(echo "$2" | cut -d'-' -f1)
+        HASH=$(echo "$2" | cut -d'-' -f2)
+        shift # past argument
+        shift # past value
+        ;;
         --no-copy)
         NO_COPY=1
         shift # past argument
         ;;
+        -q|--quiet)
+        QUIET=1
+        shift # past argument
+        ;;
```

Comment:
Quiet is harmful. Being quiet during build is a anti-goal, unless it's really really really too much.

Reply:
For revng-ui it's ~800 lines and vscode-web is a few thousands, I'd say that quieting fetch.sh is justified (Errors are still printed on stderr if something goes wrong)

Reply:
Fixed is in a slightly lateral way: since errors show up on stderr, I'm piping stdout to `/dev/null`
So as a result the `-q` option can be dropped